index.html

```diff
@@ -71,210 +69,48 @@ <h1>
-                <div id="s4div1">
-                    <div id="s4imgm" class="cardimgm">
-                        <img src="./images/s4.svg" alt="A snapshot of the multi-post stories project">
-                    </div>
-                    <div id="s4imgd" class="cardimgd">
-                        <img src="./images/s4imgd.svg" alt="A snapshot of the uber navigation project">
-                    </div>
-                </div>
-                <div id="s4div2">
-                    <div id="s4text" class="cardtext">
-                        <h2 class="s4h2m">Multi-Post <br> Stories</h2>
-                        <h2 class="s4h2d">Uber Navigation</h2>
-                        <div class="info">
-                            <h3 class="s4h4m">CANOPY</h3>
-                            <h3 class="s4h4d">Uber</h3>
-                            <ul id="s4ulm" class="extra">
-                                <li id="s4li1m"><span>Back End Dev</span></li>
-                                <li id="s4li2m"><span>2015</span></li>
-                            </ul>
-                            <ul id="s4uld" class="extra">
-                                <li id="s4li1d"><span>Back End Dev</span></li>
-                                <li id="s4li2d"><span>Full Stack Dev</span></li>
-                                <li id="s4li3d"><span>Lead Developer</span></li>
-                                <li id="s4li4d"><span>2015</span></li>
-                                <li id="s4li5d"><span>2018</span></li>
-                            </ul>
+                        <div>
+                            <a><button></button></a>
+                            <img>
                         </div>
-                        <p class="s4pm">A daily selection of privately personalized reads; no accounts or sign-ups
-                            required.
-                        </p>
-                        <p class="s4pd">A smart assistant to make driving more safe, efficient, and fun by unlocking
-                            your
-                            most expensive computer: your car.</p>
-                        <ul class="skills">
-                            <li>
-                                <a href="#"><button class="b1">html</button></a>
-                            </li>
-                            <li>
-                                <a href="#"><button class="b6">Ruby on rails</button></a>
-                            </li>
-                            <li>
-                                <a href="#"><button class="b2">css</button></a>
-                            </li>
-                            <li>
-                                <a href="#"><button class="b3">javaScript</button></a>
-                            </li>
-                        </ul>
-                        <a href="#"><button class="b4">See Project</button></a>
                     </div>
                 </div>
             </article>
-        </section>
+        </template>
```

Comment:
* Kindly adjust the popup window for the desktop device  you can reduce it as shown in the design so that a user doesn't have to scroll all the way down to see the popup
* ## Yours
![image](https://user-images.githubusercontent.com/52497246/226063981-76eb457b-a348-4046-a456-c0490316328a.png)
## Design 
![image](https://user-images.githubusercontent.com/52497246/226064021-125365ca-7e9a-424e-892f-b36d4548ebf1.png)

Reply:
Done @Davidosky007!  I've finished with the required changes.